plugins/wasm-go/extensions/ai-cache/core.go

```diff
@@ -164,8 +167,16 @@ func handleQueryResults(key string, results []vector.QueryResult, ctx wrapper.Ht
 			// direct return the answer if available
 			processCacheHit(key, mostSimilarData.Answer, stream, ctx, config, log)
 		} else {
-			// otherwise, continue to check cache for the most similar key
-			CheckCacheForKey(mostSimilarData.Text, ctx, config, log, stream, false)
+			// // otherwise, continue to check cache for the most similar key
+			// err = CheckCacheForKey(mostSimilarData.Text, ctx, config, log, stream, false)
+			// if err != nil {
+			// 	log.Errorf("check cache for key: %s failed, error: %v", mostSimilarData.Text, err)
+			// 	proxywasm.ResumeHttpRequest()
+			// }
+
+			// Otherwise, do not check the cache, directly return
+			log.Warnf("No cache hit for key: %s, however, no answer found in vector database", mostSimilarData.Text)
```

Comment:
哦，这个写的有问题，主要是匹配到向量数据库的key，但是对应的结果没有记录在向量数据库并且也没有配置缓存数据库的情况。
```go
// Otherwise, do not check the cache, directly return
log.Warnf("Cache hit for key: %s, but no corresponding response found in the vector database", mostSimilarData.Text)
proxywasm.ResumeHttpRequest()
```